Makefile

```diff
@@ -68,7 +68,8 @@ checklicenses: requirements.txt ## check dependencies meet licence rules
 	poetry run liccheck -s liccheck.ini
 
 test: install ## run tests quickly with the default Python
-	poetry run pytest --verbose --capture=no
+	poetry run pytest tests/test_libgsf_load.py --verbose --capture=no
+	poetry run pytest --deselect=tests/test_libgsf_load.py --verbose --capture=no
```

Comment:
Some quirk of pytest around mocking environment variables means that the test_libgsf_load tests have to be run first. S